include/deal.II/base/mpi_consensus_algorithms.templates.h

```diff
@@ -1,925 +0,0 @@
-// ---------------------------------------------------------------------
-//
-// Copyright (C) 2011 - 2022 by the deal.II authors
-//
-// This file is part of the deal.II library.
-//
-// The deal.II library is free software; you can use it, redistribute
-// it, and/or modify it under the terms of the GNU Lesser General
-// Public License as published by the Free Software Foundation; either
-// version 2.1 of the License, or (at your option) any later version.
-// The full text of the license can be found in the file LICENSE.md at
-// the top level directory of deal.II.
-//
-// ---------------------------------------------------------------------
-
-#ifndef dealii_mpi_consensus_algorithm_templates_h
```

Comment:
Could we keep this file for now (empty) and instead add a warning?

Reply:
Yes, great suggestion. I should have thought of that before.